website_contact_lastname/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Website Contact Lastname",
+    "version": "15.0.1.0.0",
+    "category": "Website",
+    "website": "https://github.com/OCA/website",
+    "author": "Sygel Technology," "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "auth_signup",
+        "website_account_fiscal_position_partner_type",
+        "partner_firstname",
```

Comment:
I'd expect this module to be named as a derivative of this :) (maybe website_sale_partner_firstname)

Reply:
done